modules/pwa/i18n.ts

```diff
@@ -71,7 +71,7 @@ export async function createI18n(): Promise<LocalizedWebManifest> {
 
   const manifestEntries: Partial<ManifestOptions> = {
     scope: '/',
-    id: '/',
+    id: 'zone.elk',
```

Comment:
Should this be `elk.zone`?

Reply:
ok, using `elk.zone`, I'll check FF on Android... Samsung seems to be broken